python/pyspark/rdd.py

```diff
@@ -296,7 +296,7 @@ def flatMap(self, f, preservesPartitioning=False):
 
         >>> rdd = sc.parallelize([2, 3, 4])
         >>> sorted(rdd.flatMap(lambda x: range(1, x)).collect())
-        [1, 1, 1, 2, 2, 3]
+        [1, 1, 1, 2, 2, 3, 3, 3, 4, 4, 4, 4]
```

Comment:
I think that the purpose of this change was to induce a test failure to verify that the output reporting was correct.